test/e2e/node/events.go

```diff
@@ -95,7 +95,7 @@ var _ = SIGDescribe("Events", func() {
 		var events *v1.EventList
 		// Check for scheduler event about the pod.
 		ginkgo.By("checking for scheduler event about the pod")
-		framework.ExpectNoError(wait.Poll(time.Second*2, time.Second*60, func() (bool, error) {
+		framework.ExpectNoError(wait.Poll(framework.Poll, 5*time.Minute, func() (bool, error) {
```

Comment:
This is a large step up (1->5m), for the future if we've only seen a slight regression in timing, it would be good to bump by a smaller threshold so we can perhaps dig deeper into why this has gotten slower.

Reply:
it can take until 6mins by design https://github.com/kubernetes/kubernetes/pull/106508#issuecomment-972264043